server/src/utilities/generateComponent/helpers/generateReact.ts

```diff
@@ -0,0 +1,64 @@
+import { Node, SyntaxKind, SourceFile } from 'ts-morph';
+
+import MezzuriteComponent from '../../../models/MezzuriteComponent';
+
+function generateReact (filePath: string, sourceFile: SourceFile): MezzuriteComponent {
+  let component = null;
+
+  if (filePath != null && sourceFile != null) {
+    const exportContents = getExportContents(sourceFile);
+
+    if (exportContents != null) {
+      let hasWithMezzurite = false;
+      let name = null;
+
+      if (exportContents.getKind() === SyntaxKind.Identifier) {
+        name = exportContents.getText();
+      } else {
+        const identifiers = exportContents.getDescendantsOfKind(SyntaxKind.Identifier);
+        const lastIdentifier = identifiers[identifiers.length - 1];
+
+        if (lastIdentifier != null) {
+          name = lastIdentifier.getText();
+        }
+
+        if (exportContents.getText().indexOf('withMezzurite') > -1) {
```

Comment:
The output value of the condition in this `if(...)` can be assigned directly to `hasWithMezzurite`. When it evaluates to `true`, it's essentially doing: `if (true) { x = true; }`.